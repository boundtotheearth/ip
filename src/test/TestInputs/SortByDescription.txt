todo ccc
todo aaa
todo bbb
sort description
