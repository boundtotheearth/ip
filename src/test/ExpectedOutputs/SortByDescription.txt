Added: [T][✘] ccc
Added: [T][✘] aaa
Added: [T][✘] bbb
Here are your tasks sorted by description
1. [T][✘] ccc
2. [T][✘] bbb
3. [T][✘] aaa
